### PR TITLE
Unified API Disclaimer

### PR DESCRIPTION
**Pull request by smeubank**
With the work on hubs and scopes refactoring, this page needs a disclaimer even more than before 

https://develop.sentry.dev/sdk/hub_and_scope_refactoring/

### CODE REVIEW

File: src/docs/sdk/unified-api/index.mdx [hunk 1]
```diff
@@ -2,6 +2,20 @@
 title: 'Unified API'
 ---
 
+<Alert title="Note" level="info">
+
+Everything written here sounds real nice. However, we live in a practical world, and expect each platform to adhere to what makese sense for the developers there. What is written below is not to be taken as strict rules. It is a guide and to be thought of as a tool to help developers build SDKs which work well which eachother.
+
+The top priority when building SDKs is a high quality API and developer experience, don’t let unified API be an excuse to build things which don't make sense for a give language.
```
Comment (adinauer) [inline src/docs/sdk/unified-api/index.mdx:9]:
```suggestion
The top priority when building SDKs is a high quality API and developer experience, don’t let unified API be an excuse to build things which don't make sense for a given language.
```

**Review by adinauer — APPROVED** [review]
LGTM

**Review by adinauer — APPROVED** [review]
LGTM

**Review by antonpirker — APPROVED** [review]
looking good!

**Pull request merged**